ios/Sources/TimetableFeature/TimetableDetailScreen.swift

```diff
@@ -0,0 +1,245 @@
+                    .bold()
+                    .foregroundColor(AssetColor.Base.primary.color)
+                Text("みなさんは、AndroidのUIの関連の状態管理や非同期処理について、CoroutiensのrepeatOnLifecycleまでそのメリット、デメリットまで含めて追えていますか？以下のQiitaではAndroidのライフサイクルの基礎からKotlin Coroutinesでのsuspend functionによる処理までは紹介しました。https://qiita.com/takahirom/items/3f012d46e15a1666fa33しかし、現在はKotlin Coroutines Flow、StateFlow、今はさらにlaunchWhenStarted、repeatOnLifecycleが登場して、少し混乱しているのではないでしょうか？ここまでの流れをまとめて紹介していきたいです。") // TODO: FIXME
+                    .font(.body)
+                    .foregroundColor(AssetColor.Base.secondary.color)
+                    .lineLimit(nil)
+            }
+        }
+        .padding(.bottom, 84)
+    }
+
+    func fixedFooter(tapSlide: @escaping () -> Void, tapVideo: @escaping () -> Void) -> some View {
+        VStack {
+            Spacer()
+
+            HStack {
+                VStack {
+                    AssetImage.iconClip.image
+                        .foregroundColor(AssetColor.primary.color)
+                        .frame(width: Const.footerIconSize, height: Const.footerIconSize)
+                }
+                .padding()
+                .background(AssetColor.primaryPale.color)
+                .cornerRadius(6)
+
+                HStack(spacing: 10) {
+                    AssetImage.iconPlayCircleFilled.image
+                        .foregroundColor(.white)
+                        .frame(width: 24, height: 24)
+
+                    Text(L10n.TimetableScreen.Detail.watchThisVideo)
+                        .font(.headline)
+                        .bold()
+                        .foregroundColor(.white)
+
+                    Spacer()
+
+                    AssetImage.iconChevron.image
+                        .renderingMode(.template)
+                        .foregroundColor(.white)
+                        .frame(width: Const.footerIconSize, height: Const.footerIconSize)
+                }
+                .padding()
+                .background(AssetColor.primary.color)
+                .cornerRadius(6)
+            }
+            .padding(8)
+            .background(AssetColor.Background.primary.color)
+        }
+    }
+}
+
+private enum DesciptionType: CaseIterable {
+    case schedule
+    case length
+    case category
+    case language
+    case difficulty
+
+    var image: SwiftUI.Image {
+        switch self {
+        case .schedule:
+            return AssetImage.iconSchedule.image
+        case .length:
+            return AssetImage.iconTime.image
+        case .category:
+            return AssetImage.iconCategory.image
+        case .language:
+            return AssetImage.iconLanguage.image
+        case .difficulty:
+            return AssetImage.iconDifficulty.image
+        }
+    }
+
+    func value(timetable: TimetableItem) -> String {
+        switch self {
+        case .schedule:
+            return "\(timetable.startsAt.formatToDate) \(timetable.startsAt.formatToTime) ~ \(timetable.endsAt.formatToTime)"
```

Comment:
In design, I think `endsAt` isn't needed.

If it is needed,  how about use `DateIntervalFormatter` ?

ref: 
- https://developer.apple.com/documentation/foundation/dateintervalformatter
- https://speakerdeck.com/treastrain/easily-and-correctly-format-any-information-into-a-string?slide=67

Reply:
I think the design is luck of this info. Android design has.